src/main/game_save.cpp

```diff
@@ -51,18 +53,60 @@ std::string makeGameSavePath(std::string episode, std::string world, std::string
     return ret;
 }
 
+static void s_LoadCharacter(SavedChar_t& dest, const saveCharState& s)
+{
+    dest = SavedChar_t();
+
+    if((s.state < 1) || (s.state > 10))
```

Comment:
Maybe put the "max state" constant rather than using the 10 magic number? I mean, for the future case when we add more state slots (for example, SMBX-38A's).

Reply:
Very good idea, I think this was used elsewhere in our code but I'll update them both.